ui/src/app/base/components/FormikFormContent/FormikFormContent.tsx

```diff
@@ -158,7 +158,7 @@ const FormikFormContent = <V, E = null>({
           {nonFieldError}
         </Notification>
       )}
-      {children}
+      {typeof children === "function" ? children({ values }) : children}
```

Comment:
This allows using render props (just like regular Formik component does)

Reply:
If we want full compatibility with using render props we might as well include the entire formik context instead of just `values`?

Reply:
I've left a comment below about using `useFormikContext` instead of an inner function.